amethyst_network/src/lib.rs

```diff
@@ -28,6 +28,28 @@ mod network_socket;
 mod server;
 mod test;
 
+/// Each event type is either reliable or unreliable:
+/// Reliable events always reach their destination,
+/// Unreliable events may be lost
+/// For Amethyst-defined events, whether it's reliable is specified in this function,
+/// Otherwise, it's specified by the use of NetEvent::Reliable vs NetEvent::Unreliable
+fn is_reliable<T>(event: NetEvent<T>) -> bool {
```

Comment:
this should be moved to an fn under `NetEvent`

Reply:
and just to clairfy this is a fn that takes a `NetEvent` which means to call this it must be imported manually and then called with a netevent. In this case its easier for the user to have a netevent and called `my_netevent.is_reliable()`.